examples/map_buffers.rs

```diff
@@ -0,0 +1,176 @@
+extern crate ocl;
+extern crate ocl_core;
+
+use std::mem;
+
+const BUFFER_DIMENSIONS: usize = 2 << 20;
+const PLATFORM_ID: usize = 0;
```

Comment:
In case you didn't know (which is likely since it's not really documented anywhere), you can set the default platform with the environment variable `OCL_DEFAULT_PLATFORM_IDX` which can be queried by calling `ocl::core::default_platform_idx()` and is used by `ocl::Platform::default()`. Using that mechanism would be preferable to setting a separate constant.